assets/src/ba_data/python/bastd/actor/spaz.py

```diff
@@ -1052,21 +1052,21 @@ def handlemessage(self, msg: Any) -> Any:
                 self.node.handlemessage('flash')
 
                 # If we're holding something, drop it.
-                if damage > 0.0 and self.node.hold_node:
+                if damage >= 0.0 and self.node.hold_node:
```

Comment:
Could you also tell something about these `> 0` -> `>= 0` changes? Seems like this things (as `damage` and `self.hitpoints`) can't be negative, so that's always evaluated as True.